tools/devtool

```diff
@@ -449,6 +449,21 @@ cmd_build() {
     # Cargo uses the debug profile by default. If we're building the release
     # binaries, we need to pass an extra argument to cargo.
     cargo_args=("$@")
+    
+    # Add the default target if we did not get that argument in the build command.
+    add_default_target=true
+    for flag in "${@}"
+    do
+        if [ $flag = "--" ]; then
+            break
+        elif [ $flag == "--target" || $flag =~ "--target=.*" ]; then
```

Comment:
This is an invalid test construct. `[` doesn't support logical operators or regex matching operators. You were probably looking for `[[`.